src/bar.jl

```diff
@@ -48,6 +48,7 @@ grouped_xy(y::AbstractMatrix) = 1:size(y,1), y
     fillrange := if isstack
         # shift y/fillrange up
         y = copy(y)
+        y[isnan.(y)] = 0
```

Comment:
Perhaps better `!isfinite.(y)`?

Reply:
That works, though the deprecation warning said to do `y[.!isfinite.(y)] = 0`. I'll update the PR.